airflow/migrations/utils.py

```diff
@@ -35,7 +35,8 @@ def get_mssql_table_constraints(conn, table_name):
      FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS AS tc
      JOIN INFORMATION_SCHEMA.CONSTRAINT_COLUMN_USAGE AS ccu ON ccu.CONSTRAINT_NAME = tc.CONSTRAINT_NAME
      WHERE tc.TABLE_NAME = '{table_name}' AND
-     (tc.CONSTRAINT_TYPE = 'PRIMARY KEY' or UPPER(tc.CONSTRAINT_TYPE) = 'UNIQUE')
+     (tc.CONSTRAINT_TYPE = 'PRIMARY KEY' or UPPER(tc.CONSTRAINT_TYPE) = 'UNIQUE'
+     or UPPER(tc.CONSTRAINT_TYPE) = 'FOREIGN KEY')
```

Comment:
One of these is not upper-cased! But do you really need `UPPER`? From what I remember, other migrations don’t do this amd still work OK.

Reply:
I don't think there's any other migration that tried dropping an unnamed foreign key in `MSSQL`. Most other foreign keys are named and easily dropped. This particular case is one of those unnamed foreign keys. There's no other way to drop unnamed foreign key in MSSQL other than using this query or using sqlalchemy `inspect`.

For the one that's not uppercased, it has been like that and working. My addition to the code is that of foreign key.

Reply:
Hmm, and the other two can be in lower case? I would imagine we should be able to get rid of those `UPPER`;

Reply:
Yea. Possible but it'll require editing released migrations. Where ever it was used, it compares to an upper case value. 
In my opinion we should keep it with the `UPPER` case